admin/sql/CreateTables.sql

```diff
@@ -953,15 +953,56 @@ CREATE TABLE genre ( -- replicate (verbose)
     last_updated        TIMESTAMP WITH TIME ZONE DEFAULT NOW()
 );
 
+CREATE TABLE genre_alias_type ( -- replicate
+    id                  SERIAL, -- PK,
+    name                TEXT NOT NULL,
+    parent              INTEGER, -- references genre_alias_type.id
+    child_order         INTEGER NOT NULL DEFAULT 0,
+    description         TEXT,
+    gid                 uuid NOT NULL
+);
+
 CREATE TABLE genre_alias ( -- replicate (verbose)
-    id                  SERIAL,
+    id                  SERIAL, --PK
     genre               INTEGER NOT NULL, -- references genre.id
     name                VARCHAR NOT NULL,
     locale              TEXT,
-    edits_pending       INTEGER NOT NULL DEFAULT 0 CHECK (edits_pending >= 0),
```

Comment:
micro-nit, but we can probably keep the space after `=` if that's the only change on this line (I realize this was probably copied from another entity)